README.md

```diff
@@ -41,77 +41,58 @@ Curious to see how it all works? Check out our [control flow diagrams](/docs/des
 
 Ready to quickly deploy the latest version of Azure Service Operator on your Kubernetes cluster and start exploring? Follow these steps.
 
-1. Make sure `kubectl` is configured to connect to the Kubernetes cluster you want to deploy Azure Service Operator to.
-    
-    To connect to an Azure Kubernetes Service cluster, you can use the below command:
 
-    ```console
-    az aks get-credentials -g <AKSClusterResourceGroup> -n <AKSClusterName>
-    ```
-
-1. Install cert-manager on the cluster using the following commands.
+1. Install [cert-manager](https://cert-manager.io/docs/installation/kubernetes/) on the cluster using the following command.
 
     ```console
```

Comment:
This is a small thing but suggest using `sh` instead of `console` when the input is a command, since `console` is for a whole shell session, it highlights the `$ cmd....` and output differently.

```suggestion
    ```sh
```